### DISCUSSION TITLE
Combining arrays from multiple sensors in an apex chart?

### DISCUSSION THREAD

**Discussion by deltamelter** [BottlecapDave/HomeAssistant-OctopusEnergy#576] [General]
Now that price is split into previous, current and next arrays in their own event entities, does anybody know of a method to combine values into a single array?  
The apex chart will auto scale according to the first entity listed in the series, so if the first entity has fewer values which are from a smaller range, the later entities get cropped.
I've tried to get the apex chart card to deal with this, but couldn't get it to work and end up using an adjustable max (`'|+5|'`) which still works on the 1st entity and works fine until the difference (eg next day 60p agile peak) is more than that adjustable max.

**Comment by pcb1962** [comment]
I'd also like to know how to do this, I added a second entity and data_generator but the effect is not quite what I was hoping for...

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/5259660/5ea1155c-c8e4-4bd1-b915-793608528b17)
